reference/HA.md

```diff
@@ -0,0 +1,12 @@
+## High Availability 
+
+Mayastor 2.0 brings in the High Availability (HA) feature. In the event of a failure, this feature will guarantee I/O continuity. 
+The HA feature consists of two components: the HA node agent and the cluster agent. The HA node agent looks for io-path failures from applications to their corresponding targets. If any such broken path is encountered, the HA informs the cluster agent. The cluster-agent then creates a new target on a different (live) node. Once the target is created, the `csi-node` establishes a new path between the application and its corresponding target. The HA feature restores the broken path within seconds, ensuring negligible downtime. 
```

Comment:
`the HA informs` -> `the HA node agent informs`.

Reply:
Updated the text. Thanks.